suites/pacific/rbd/tier-2_rbd_mirror_regression.yaml

```diff
@@ -239,3 +239,14 @@ tests:
             io-total: 200M
       polarion-id: CEPH-9521
       desc: Testing mirroring of cloned image
+
+  - test:
+      name: Mirroring from journal to snapshot
+      module: test_rbd_mirror_journal_to_snap.py
+      clusters:
+        ceph-rbd1:
+          config:
+            imagesize: 2G
+            io-total: 200M
```

Comment:
I don't see this module being reused for other purpose than this image and these image size and io-total being used same values and overriden in module
I think these configs can be removed and focused on handling with defaults in testcase module.

Reply:
Done.